### PR TITLE
fix: Resolver bugfixes

### LINKED ISSUE #6288: Every SolidJS Example is Suddently broken
# 🐛 bug report

## Preflight Checklist

- [x] I have read the [Contributing Guidelines][contributing] for this project.
- [x] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem

Not sure what happened but every [SolidJS](https://github.com/solidjs/solid) example stopped working a day or so ago. There are varying errors from issues with babel:

> babel-transpiler.88c24584.worker.js:2015 Error: Cannot find module 'babel-preset-solid' from 'babel-preset-solid'

To non-compiled examples complaining that imports aren't functions.

## How has this issue affected you? What are you trying to accomplish?

These examples have been up for years, and have been our main online examples historically. Some people in the community started mentioning our examples were broken, and this is what I encountered.

### Link to sandbox:

[Hello World Template](https://codesandbox.io/s/solid-hello-world-9v13b)
[Non-Compiled Tagged Template Literals](https://codesandbox.io/s/solid-tagged-template-literal-example-btgbk)
[Only Reactivity](https://codesandbox.io/s/reaction-5wyzu)

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |              |
| Browser          |  Chrome   |
| Operating System | MacOS  BigSur  |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by DeMoorJasper**
## What kind of change does this PR introduce?

- Add cjs to react preset v1
- Fix some more issues in babel plugin/preset installer
- Support more complex exports fields

Closes #6288

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6285/commits/97492f4b09578b980433ed005aabd45d8a9196e5 is at https://pr6285.build.csb.dev/s/new.

File: packages/app/src/sandbox/eval/transpilers/babel/index.ts [hunk 1]
```diff
@@ -151,10 +151,11 @@ class BabelTranspiler extends WorkerTranspiler {
...
       } catch (err) {
-        console.warn(
-          `Error occurred while trying to quickly transform '${path}'`
-        );
-        console.warn(err);
+        // We should log this to sentry or something instead, it confuses users...
+        // console.warn(
+        //   `Error occurred while trying to quickly transform '${path}'`
+        // );
+        // console.warn(err);
```
Comment (CompuIves) [inline packages/app/src/sandbox/eval/transpilers/babel/index.ts:158]: Should these be gone?

Comment (CompuIves) [reply packages/app/src/sandbox/eval/transpilers/babel/index.ts:158]: Ah I now see the comment. Maybe we should wrap this in an `if NODE_ENV === development` for now

**Review by CompuIves — APPROVED** [review]
LGTM!@

**Pull request merged**